### PR TITLE
Add example where inputs are hidden after successfull submission.

### PR DESCRIPTION
**Pull request by kul3r4**
Add example where inputs are hidden after successfull submission.
This could be further improved introducing check/fail icons, but needs further investigation because the img is not always showing. In the meanwhile, this will address the request of hiding the inputs when a form is successfull.

### CODE REVIEW

File: src/20_Components/amp-form.html [hunk 1]
```diff
@@ -71,5 +74,23 @@
         </template>
     </div>
   </form>
+
```
Comment (sebastianbenz) [inline src/20_Components/amp-form.html]: add a heading for the sample

File: src/20_Components/amp-form.html [hunk 2]
```diff
@@ -71,5 +74,23 @@
...
+  <!--
+    Use the `amp-form-submit-success` class to hide inputs after a successfull submission.
```
Comment (sebastianbenz) [inline src/20_Components/amp-form.html]: s/inputs/input fields

Comment (sebastianbenz) [inline src/20_Components/amp-form.html]: You could embed the corresponding css code:

This hides all form input fields after successful form submission:
```
form.amp-form-submit-success.hide-inputs > input {
    display: none
}
```

File: src/20_Components/amp-form.html [hunk 3]
```diff
@@ -71,5 +74,23 @@
...
+  -->
+  <form class="hide-inputs" method="post" action="/components/amp-form/submit-form" action-xhr="/components/amp-form/submit-form-xhr" target="_top">   
+    <input type="text" name="name" placeholder="Name..." required>
+    <input type="submit" value="Subscribe" class="button button-primary">
+    <div submit-success>
+        <template type="amp-mustache">
```
Comment (sebastianbenz) [inline src/20_Components/amp-form.html]: make the sample simpler: remove templates and simply write success or error:

```
<div submit-success>Success!</div>
<div submit-error>Error!</div>

**Comment by sebastianbenz** [comment]
few comments

**Comment by kul3r4** [comment]
@sebastianbenz PTAL

File: src/20_Components/amp-form.html [hunk 4]
```diff
@@ -60,6 +63,23 @@
       <input type="email" name="email" placeholder="Email..." required>
       <input type="submit" value="Subscribe" class="button button-primary">
 
+    <div submit-success>Success!</div>
+    <div submit-error>Error!</div>
+  </form>
+
+  <!-- #### Hiding input fields after success -->
+  <!--
+  Use the `amp-form-submit-success` class to hide input fields after a successfull submission. This hides all form input fields after successful form submission:
```
Comment (sebastianbenz) [inline src/20_Components/amp-form.html]: s/This hides /The following CSS rule hides/

File: src/20_Components/amp-form.html [hunk 5]
```diff
@@ -60,6 +63,23 @@
       <input type="email" name="email" placeholder="Email..." required>
       <input type="submit" value="Subscribe" class="button button-primary">
 
+    <div submit-success>Success!</div>
```
Comment (sebastianbenz) [inline src/20_Components/amp-form.html]: Is it possible that you've removed amp-mustache from the wrong sample?

Comment (kul3r4) [inline src/20_Components/amp-form.html]: @sebastianbenz Yes I did! Fixed it!

**Comment by sebastianbenz** [comment]
LGTM

**Pull request merged**